config/config.go

```diff
@@ -12,6 +12,10 @@ import (
 	"github.com/spf13/viper"
 )
 
+var (
+	defaultBaseURL = "https://v2.exercism.io/api/v1"
```

Comment:
I find this to be a bit confusing seeing as config/config is a generic wrapper around config with no specific exercism data. In my opinion it is probably better off in `user_config.go`

Reply:
Yepp, that's a good point. I'll make that change.